### PR TITLE
Issue #2533: Update comment error messages

### PR DESCRIPTION
**Pull request by AnneTee**
@kostajh This updates the comment form error messages to be a little more specific/polite and adds a variable to tell the website which field is causing the error so I can add a class to it and style it.

This goes along with [this PR](https://github.com/savaslabs/savaslabs.github.io/pull/210) for the company site. 🌈

### CODE REVIEW

File: source/app/Http/Middleware/ValidInputMiddleware.php [hunk 1]
```diff
@@ -21,29 +21,34 @@ public function handle($request, Closure $next)
...
+            $responseParams = array('message' => 'Please include a comment.', 'status' => 400);
         }
         else if (!$request->input('slug')) {
             // TODO: Check against website to ensure the slug is valid.
             $responseParams = array('message' => 'Slug is required', 'status' => 400);
         }
         else if (!$request->input('nocaptcha')) {
-            $responseParams = array('message' => 'No captcha response required', 'status' => 400);
+            $responseParams = array(
+              'message' => 'Please answer "What type of animal is the Savas logo?"',
```
Comment (kostajh) [inline source/app/Http/Middleware/ValidInputMiddleware.php]: @AnneTee should `Savas` be `Savas Labs`?

Comment (AnneTee) [reply source/app/Http/Middleware/ValidInputMiddleware.php]: @kostajh Good call, pushed the fix!

**Comment by AnneTee** [comment]
@kostajh I made the update and the unit tests are passing now, but some of the Behat test are still failing. I updated the error message text in the Behat tests to match the recently updated error messages, but I'm still seeing some failing tests because the `data` array appears to be empty when it should contain formatted comment data.

For example, for `Scenario: Get all comments via the API` I get the error:

```
Failed asserting that '{"success":true,"data":[],"message":""}' matches PCRE pattern "/A Comment\!/i".
```

I'm not sure why the `data` array is being set to empty for successful comments.

p.s. definitely need some commit squashing here 😂 😹 :yey:

**Comment by AnneTee** [comment]
Closing in favor of #20

**Pull request closed without merging**